ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -36,32 +39,46 @@ component:
   guidevane: "Control vanes that meter refrigerant (on a centrifugal chiller) or air (on an AHU)."
   heater: "Component which provides heat to media."
   humidifier: "Component which humidifies."
+  interface: “A device/point which enables 2 systems/devices to communicate” # added 25/08/20
+  meter: "A device that measures and records the quantity, degree, or rate of something." # Added 10/09/20
+  module: "One of a set of separate parts that, when combined, form a complete whole device/system." # Added 05/10/20
   pump: "Component used for the distribution of liquid media."
+  rack: "Component used in comms rooms for mounting servers & switches" # Added 25/08/20
   shade: "Window covering."
-  tank : "Storage device for media."
+  tank: "Storage device for media."
```

Comment:
would it be correct to limit this to liquid media?

Reply:
In theory, "tank" could also apply to compressed air (used for some high-performance actuators in data centers) or diesel fuel for day tanks wherever you have a generator.